ingest/src/main.ts

```diff
@@ -1,4 +1,4 @@
-import * as yargs from "yargs";
+import yargs = require("yargs");
```

Comment:
??

is this a CSJ thing?

Reply:
Yeah, leftovers from the ESM rabbit hole